development/native/win_10.md

```diff
@@ -0,0 +1,200 @@
+    ```
+
+* Enable Virtual Machine Platform
+
+    ```.bash
+    dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart
+    ```
+    Restart your computer.
+
+* Setup WDL v2 as default
+
+    ```.bash
+    wsl --set-default-version 2
+    ```
+
+## Install Linux Distro
+
+There are several Linux distros available for WSL2, in our example we
+use Ubuntu 20.04.
+
+Go to `Microsoft Store` app, search for Ubuntu, install and launch it. You will
+see it appearing in a separate terminal this time. You can customize it right
+in the terminal, or you can close it, and reopen it in `Windows Terminal`,
+where it will appear now as an option.
+
+Once Ubuntu 20.04 is installed, open it in the Windows Terminal, run:
+
+```bash
+sudo apt update
+sudo apt dist-upgrade
+```
+
+## Install Postgresql database
+
+Open Ubuntu bash in Windows Terminal
+
+Add PostgreSQL source repository for apt-get.
+
+```
+echo "deb [arch=amd64] http://apt.postgresql.org/pub/repos/apt/ focal-pgdg main" | sudo tee -a /etc/apt/sources.list.d/pgdg.list
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+sudo apt-get update
+```
+
+Install required packages.
+```
+sudo apt-get install -y postgresql-12 postgresql-contrib-12 libgeos-dev libproj-dev postgresql-12-postgis-3 postgresql-12-postgis-3-scripts libpq-dev cmake imagemagick libmagickwand-dev tesseract-ocr git meld curl
+```
+check postgresql version:
+
+```bash
+psql --version
+```
+
+When prompted do not supply a password. See below, the password must match `config/database.yml` if provided.
+```
+sudo service postgresql start
+sudo -u postgres createuser -s -d -P taxonworks_development
+```
+
+Change permissions for posgresql, you are changing 'peer' to 'trust' for the matched line.
+```
+sudo sed -i.bak 's/local\s*all\s*all\s*peer/local all all trust/'  /etc/postgresql/12/main/pg_hba.conf
+sudo service postgresql restart
+```
+
+
+Now you should be able to start postgres, stop, ask for a status:
+
+```bash
+sudo service postgresql status
+sudo service postgresql start
+sudo service postgresql restart
+sudo service postgresql stop
+```
+
+Configure apt-get to point to newer Node packages
+```
+curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
+sudo apt-get install -y build-essential nodejs
+sudo apt autoremove
+```
+
+Install virtual ruby environment. You can follow [Ubuntu 20.04 instructions](ubuntu_20_04.md) for
+for installing `rvm`. Here I use an alternative `rbenv` package, which is more lightweight.
+
+## Install rbenv
+
+Rbenv, like rvm, allows to run several versions of ruby at the same time.
+
+```bash
+git clone https://github.com/sstephenson/rbenv.git ~/.rbenv
+git clone https://github.com/sstephenson/ruby-build.git ~/.rbenv/plugins/ruby-build
+git clone https://github.com/rkh/rbenv-update.git ~/.rbenv/plugins/rbenv-update
+```
+
+to `~/.bashrc` add line:
+
+```bash
+eval "$(rbenv init -)"
```

Comment:
@dimus when doing this and restarting the terminal first line reads `Command 'rbenv' not found`. Per https://github.com/rbenv/rbenv#basic-github-checkout looks we need `echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc` here.

Reply:
good catch